### DISCUSSION TITLE
COINS STOLEN FROM CHIA CLIENT

### DISCUSSION THREAD

**Discussion by Bulat1795** [Chia-Network/chia-blockchain#10691] [Support]
I'm desperately asking for help from the developers. 6.25 chia coins were stolen from my wallet in the chia client (version 1.2.11). I have been saving them and mining for 6 months and have just started to recoup my investment in equipment. The coins were withdrawn to the scammer's wallet without my knowledge. This is a fatal mistake of the chia developers, because they cannot guarantee the security of my wallet. I very much ask you to refund me my coins. Fraudster wallet number: xch1sq68hqkthcpjrz0eddeesum9pg2chkpxhgygh7g45mdplkz6c66s8sx4ma

In addition, I want to say that I am an active member of the community and participate in the development of the chia coin mining pool
![Снимок экрана 2022-03-12 в 18 15 17](https://user-images.githubusercontent.com/101464750/158023760-9aca56ee-1b11-4c44-bde7-10c5320302bd.png)

**Comment by lincolnthalles** [comment]
Chia won't refund you, that's not how the world works.

You probably executed some sketchy tool that stolen your keys. You may still have malware installed in your system.
I suggest you to reinstall the operating system from all PCs that have access to this wallet and setup everything again.
You may backup `.chia` and `.chia_keys` folder from the main node to speed-up setup and going back to farming.

As **your mnemonic key was probably stolen**, and I suppose your plots are tied to this key, you may circumvert this fatal security breach by using a new, uncompromised wallet to ***receive*** your farming rewards. You may even use a xch exchange (where you can sell the coins) wallet. If you are still plotting, keep using the same compromised wallet/NFT for this. At this point, this is fine.

## Changing the farming rewards wallet
* Open Chia GUI
* Go to `Pool` tab

  For each plot NFT:
  * Click `Edit Payout Instructions` (hidden within the three vertical dots menu)
  * Paste your new and secure xch wallet and click `Save`. It may show an error or open the same window again. Just ignore and close it. Click Edit again to confirm the new wallet address was saved.

  For 0.25 XCH network rewards and OG plots farming:
  * Go to `Farm` tab
  * Click `Manage Farming Rewards` (hidden within the three vertical dots menu)
  * Paste you new wallet address on both fields and Save.
* It's important to do this on each farmer/node instance
* Any incoming rewards will go straight to the cold wallet/exchange wallet

**Remember to only access your secure wallet (a.k.a cold wallet) and exchange accounts from a safe computer, where you don't do experiments with any third-party tools, crypto related or not.**
**Do not add your secure wallet mnemonic key to any of your farming or plotting machines. You only need to paste it´s xch adress on the fields I mentioned. The only wallet key those machines need is the already compromised one, to load your existing plots and farm.**
Plotting machines are more prone to this kind of security breach as it's common to rely on third-party tools to do the hard work.

If you prefer to use another self-managed xch wallet rather than an exchange wallet, just download the new Chia 1.3.0 (use the new wallet-mode, it's fast and won't sync the huge blockchain). Do this on a **safe** computer and create a new private key/wallet. Don't add any NFTs or create plots using this key. You just need to paste the xch Receive Address on your farmers and keep this wallet 24-word mnemonic seed safe.
You may check your new wallet balance safely and easily from any computer or phone using https://xchscan.com/ or https://www.chiaexplorer.com/

**Reply by Bulat1795** [reply]
Are you seriously?  This scammer is being talked about in all communities.  Approximately every 5 minutes, he withdraws money from other people's wallets and this only happens on versions below 1.3.0.  Are you saying that everyone has compromised data or malware?  I have never entered my mnemonics anywhere and I perfectly understand what it could cost me.  Hundreds of accounts are being robbed from under your nose, and you are talking to me about personal responsibility?  The problem has become widespread and everyone is talking about it in all communities!

 I will send your answer to everyone and show how you feel about the problem.  I have never seen such an attitude towards their users anywhere.  Some guy gets rich at the expense of others, and you're talking about personal responsibility.  This makes me think it's not "some guy" Thousands of thefts are your fault.  Everyone will know about your attitude to this problem.

**Reply by lincolnthalles** [reply]
I'm not related to Chia. Everyone is free to post in github discussions and issues to try to help.
Your attitude is being worse than what you are trying to pin on me.

"I was stolen, don't know how and you must refund me" won't work.
If there is a node vulnerability or somewhat that is being explored, of course it needs discussion to address the issue, but so far I haven't seen any indications of that, nor you provided enough details that points to that.

If your systems have a security breach, you don't need to input your mnemonic key on other services for it to be stolen. It can be stolen directly from your computers. Also, transactions can be issued from your wallet the same way. All that is needed to lose your funds is to run a malicious third-party software on the same computer that has the keys. That's why I suggested a full clean-up and taking some security measures.
You can see this kind of theft with metamask and other crypto wallets. Typically, nobody should blame the victim. But so far in crypto thefts, the user always made it easy by running a malicious tool or browser extension.

Are you sure you aren't using any third-party tools to manage your wallet, farm or plotting? As I said, Chia can't do anything about it if any of your systems is compromised. At most, this scammer wallet could be blocked somehow, but the thief would just switch to another.

If this is happening to other people, they are most likely using the same compromised third-party tool or service. That is where your investigation should start. After you do some digging on your own, then you can post your findings in here to try to narrow down the issue and warn other people.

**Reply by Bulat1795** [reply]
Извини. Я думал ты из команды разработчиков или из службы поддержки. Спасибо за объяснения.

**Comment by shaneo257** [comment]
its upto the user, not chia devs on protecting your keys
this sounds like a case of bad user security practices
